README.md

```diff
@@ -2,27 +2,28 @@
 
 ## About
 
-The sambacc is an young project that aims to consolidate and coordinate
-configuration of [Samba](http://samba.org), and related components, when
-running in a container. The configuration of one or many instances can be
-provided to the tool by way of a JSON file which then handles the low level
```

Comment:
suggestion: s/JSON file/configuration file in json, yaml, or toml format/

Reply:
This comment applied to the removed text. I did look for, and find, one other use of "JSON configuration" that I updated.